src/FffGcodeWriter.cpp

```diff
@@ -1261,15 +1250,15 @@ bool FffGcodeWriter::processInsets(const SliceDataStorage& storage, LayerPlan& g
                 // nothing to do
                 return false;
             }
-            const unsigned bottom_layers = mesh.getSettingAsCount("bottom_layers");
-            if (layer_nr >= bottom_layers)
+            const int bottom_layers = mesh.getSettingAsCount("bottom_layers");
```

Comment:
The frontend says "minimum_value": "0".

The frontend does all user input checking.
I don't think we should change all that for this feature.